src/main/java/jenkins/plugins/logstash/persistence/ElasticSearchDao.java

```diff
@@ -61,7 +61,7 @@
  * @since 1.0.4
  */
 public class ElasticSearchDao extends AbstractLogstashIndexerDao {
-  final HttpClientBuilder clientBuilder;
+  transient HttpClientBuilder clientBuilder;
```

Comment:
Hmm, maybe. It definitely used to work for me, but I do not exactly recall which branch I was testing